src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+    private String deadline;
+    private LocalDate date;
+    private LocalTime time;
+    
+    public Deadline(String description, boolean isCompleted, String deadline) {
+        super(description,isCompleted);
+        this.deadline = deadline;
+
+        this.date = parseDate(deadline);
+        this.time = parseTime(deadline);
+
+    }
+    public String getDeadline() {
+        return this.deadline;
+    }
+
+    public String getTaskSymbol() {
+        return "[D]";
+    }
+
+    public String storeFormat() {
```

Comment:
Perhaps you can use a noun and something less ambiguous for the method name such as taskFormatter()?